examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Waiting for Kafka on port 9092"
+wait_for 120 sh -c "docker-compose logs -f kafka | grep -q 'started (kafka.server.KafkaServer)'"
+run_log "Kafka launched"
+
+# Topic
+TOPIC="envoy.kafka.broker"
+run_log "Creating topic"
+docker-compose exec -T kafka kafka-topics --bootstrap-server localhost:19092 --create --topic $TOPIC quickstart-events
+
+run_log "Checking topic"
+topic_created=$(docker-compose exec -T kafka kafka-topics --bootstrap-server localhost:19092 --list)
+if [[ "$topic_created" == "$TOPIC" ]]; then
+    run_log "Checked topic $topic_created succesfully"
+else
+    run_log "Checked topic $topic_created failed"
+    exit 1
+fi
+
+# Initialize message for producer
+MESSAGE="Welcome to Envoy and Kafka Broker filter!"
+
+# Producer
+run_log "Create Producer and send message"
+docker-compose exec -T kafka /bin/bash -c "echo $MESSAGE >> message.txt & kafka-console-producer --request-required-acks 1 --broker-list localhost:19092 --topic envoy.kafka.broker < message.txt"
+run_log "Sent messages succesfully"
+
+# Consumer
+run_log "Create Consumer"
+
+read_message() {
+    run_log "Reading message"
+    docker-compose exec -T kafka kafka-console-consumer --bootstrap-server localhost:19092 --topic $TOPIC --from-beginning >> consumer.txt
+}
+
+check_message() {
+    while true; do
+    if [[ $(< consumer.txt) == "$MESSAGE" ]]; then
+        run_log "Received message succesfully"
+        rm consumer.txt # clean up consumer.txt
+        run_log "Bring down the kafka"
+        docker-compose stop kafka
```

Comment:
not sure why this is here but its not the responsibility of this script to bring down the stack

Reply:
I have stuck when I want to exit 
```
docker-compose exec -T kafka kafka-console-consumer --bootstrap-server localhost:19092 --topic $TOPIC --from-beginning >> consumer.txt
```
so I use this way. Do you have any suggestions for it?

Reply:
not sure but normally exec will run a command and exit - if that is not happening then there is something wrong going on

what does the `--bootstrap-server` flag do ?

im guessing whatever is going on here should have happened in the composition

Reply:
`kafka-console-consumer` is the listener so it should be there forever to receive new messages. Let me find another way to exit the command.
`--bootstrap-server` is the server of Kafka broker and `localhost:19092` is the host and port of Envoy.

Reply:
reading kafka docs - it seems like the `--boostrap-server` is probably going to be needed - we can figure out further